app/services/moment.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+import moment from 'moment';
+
+const { computed, defineProperty } = Ember;
+
+export default Ember.Service.extend({
+  init() {
+    this._super(...arguments);
+
+    let locale = null;
+
+    defineProperty(this, 'locale', computed({
```

Comment:
Tts also worth noting, pure Object.defineProperty is like 70-100x slower then a simple property assignment. o_0). So it isn't really embers defProp alone that causes us grief...